api/librdb-api.h

```diff
@@ -207,11 +206,13 @@ typedef struct RdbHandlersDataCallbacks {
     RdbRes (*handleHashElement)(RdbParser *p, void *userData, RdbBulk field, RdbBulk elm, uint64_t totalNumElm);
     RdbRes (*handleSetElement)(RdbParser *p, void *userData, RdbBulk elm, uint64_t totalNumElm);
     RdbRes (*handleZsetElement)(RdbParser *p, void *userData, RdbBulk elm, double score, uint64_t totalNumElm);
-    /*** TODO: RdbHandlersDataCallbacks: handleStreamNewGroup, handleStreamMetadata, handleStreamElement, handleStreamConsumerPendingEntry ***/
-    RdbRes (*handleStreamNewGroup)(RdbParser *p, void *userData, RdbBulk grpName, RdbStreamID lastId);
+    /*** TODO: RdbHandlersDataCallbacks: stream stuff ... ***/
     RdbRes (*handleStreamMetadata)(RdbParser *p, void *userData, RdbStreamMeta *meta);
     RdbRes (*handleStreamElement)(RdbParser *p, void *userData, RdbStreamID *id, RdbBulk field, RdbBulk value, uint64_t totalNumElm);
-    RdbRes (*handleStreamConsumerPendingEntry)(RdbParser *p, void *userData, RdbBulk groupName, RdbStreamPendingEntry *pendingEntry);
+    RdbRes (*handleStreamNewCGroup)(RdbParser *p, void *userData, RdbBulk grpName, RdbStreamID lastId);
```

Comment:
i think that instead of a direct member for `lastId` we should have a metadata struct, so more metadata can be added later.
same for handleStreamNewConsumer with regards to `activeTime and seenTime`.